intro-HPC/ch_interactive_gent.tex

```diff
@@ -0,0 +1,54 @@
+\chapter{HPC-UGent interactive cluster}
+\label{ch:interactive_ugent}
+
+
+\section{Purpose}
+\label{sec:interactive_ugent_pupose}
+The purpose of this cluster is to give the user an environment where
+there should be no waiting in the queue to get access to a limited
+number of resources. This environment allows a user to immediatelty
+start doing, and is the ideal place for interactive work such as
+development, debugging and light prodcution (typically sufficient
+for training and/or courses). This enviroment should be seen as an
+extension of the loginnodes, instead of the a compute resource.
+The interactive cluster is overcommitted, which means that more CPU cores can be
+assigned to jobs than physically exist in the cluster. Obviously, the performance of this cluster
+heavily depends on the workloads and that actual overcommit usage. Be aware that jobs can slow
+down or speed up during their execution.
+\\ Due to the restrictions and sharing the CPU resources (see section~\ref{subsec:interactive_ugent_restrictions}) 
+jobs on slaking should normally start more or less immediately. As a consequence,
+typical workloads for this cluster should be limited to:
+\begin{itemize}
+  \item  Run interactive jobs (see chapter~\ref{ch:running-interactive-jobs}) with lots of idle time
```

Comment:
`Run interactive jobs` -> `Interactive jobs`

Reply:
i would also remove the "with lots of idle time". we stress enough that we trade interactivty for performance, and if we don't repeat it a few more times ;)